.buildkite/rllib_contrib.rayci.yml

```diff
@@ -0,0 +1,11 @@
+group: rllib contrib tests
+depends_on:
+  - oss-ci-base_build
+steps:
+  - label: ":exploding_death_star: rllib contrib: a2c tests"
+    if: build.env("BUILDKITE_PIPELINE_ID") == "0189942e-0876-4b8f-80a4-617f988ec59b"
```

Comment:
if you want, we can also add a `only-on-premerge` tag. :)

Reply:
![](https://media1.giphy.com/media/9G3xuelhlGbiyVocla/giphy.gif?cid=5a38a5a2j7j3lfdjes4np730dhzv4ff8r6r9n88c7vvyegot&ep=v1_gifs_search&rid=giphy.gif&ct=g)